src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+import duke.DukeException;
+import duke.Ui;
+import duke.Parser;
+import duke.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+import java.time.LocalDate;
+
+/**
+ * Handles Duke, a Personal Assistant Chatbot that helps a person to keep track of various things.
+ */
 public class Duke {
+    public enum TaskType {
+        TODO,
+        EVENT,
+        DEADLINE
+    }
+    private static final TaskList list = new TaskList();
+    private static final Ui ui = new Ui();
+    private static final Storage storage = new Storage(list);
+    private static final Parser parser = new Parser();
+
+    /**
+     * Adds task into task list.
+     *
+     * @param description Description of task to be added.
+     * @param taskType Task type of task to be added.
+     */
+    private static void addTask(String description, TaskType taskType) {
+        switch (taskType) {
+            case TODO:
```

Comment:
Based on the module's style guide, the `switch` and `case` statements should have the same indentation levels.